tests/basic/pure_test.go

```diff
@@ -115,3 +116,40 @@ var _ = Describe("{PureVolumeCRUDWithPXCTL}", func() {
 		AfterEachTest(contexts)
 	})
 })
+
+// This test performs a basic test making sure Volume mount options are working as expected.
+var _ = Describe("{PureFaFbValidateVolumemount}", func() {
+	var contexts []*scheduler.Context
+	var mountoption = []string{"nosuid"}
+	It("Has to inspect FA/FB  DA volumes and validate mountoptions", func() {
+		contexts = make([]*scheduler.Context, 0)
+		contexts = ScheduleApplications(fmt.Sprintf("nginx-fa-fb-davol"))
+		ValidateApplications(contexts)
+		for _, ctx := range contexts {
+			vols, err := Inst().S.GetVolumes(ctx)
+
+			if err != nil {
+				log.FailOnError(err, "Failed to get app %s's volumes", ctx.App.Key)
+			}
+			log.Infof("volumes of app %s are %s", ctx.App.Key, vols)
+
+			for _, v := range vols {
+				attachedNode, err := Inst().V.GetNodeForVolume(v, defaultCommandTimeout, defaultCommandRetry)
+				if err != nil {
+					log.FailOnError(err, "Failed to get app %s's attachednode", ctx.App.Key)
+				}
+				Step("validating mount options", func() {
+					err = Inst().V.ValidatePureFaFbMountOptions(v.ID, mountoption, attachedNode)
+					Expect(err).To(BeNil(), "Failed to validate mount options")
```

Comment:
please use dash.VerifySafely or dash.VerifyFatal instead of expect

Reply:
Done